diff --git a/node_modules/@capacitor/status-bar/Package.swift b/node_modules/@capacitor/status-bar/Package.swift
index f94fd8a..81f5792 100644
--- a/node_modules/@capacitor/status-bar/Package.swift
+++ b/node_modules/@capacitor/status-bar/Package.swift
@@ -11,14 +11,14 @@ let package = Package(
             targets: ["StatusBarPlugin"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "8.0.0")
+        .package(name: "CapacitorBinaries", path: "../../../ios/App/CapacitorBinaries")
     ],
     targets: [
         .target(
             name: "StatusBarPlugin",
             dependencies: [
-                .product(name: "Capacitor", package: "capacitor-swift-pm"),
-                .product(name: "Cordova", package: "capacitor-swift-pm")],
+                .product(name: "Capacitor", package: "CapacitorBinaries"),
+                .product(name: "Cordova", package: "CapacitorBinaries")],
             path: "ios/Sources/StatusBarPlugin"
         ),
         .testTarget(
